learning_resources/moderating-oss-repos.md

```diff
@@ -0,0 +1,342 @@
+
+### Examples
+
+#### Building and Supporting a Healthy Project Community
+
+- [x] Use people's pronouns
+- [x] Add alt text to shared images
+- [x] Ask questions, seek to understand before challenging or correcting
+- [x] Gratitude - a simple thank you goes a long way
+- [x] Patience - give time for maintainers to respond to issues, pings and the
+      like
+- [x] Celebrate each other's success
+- [x] Avoid comparison of contribution value, frequency or importance
+- [x] Remember that life happens, sometimes people are away for personal
+      reasons, and not ignoring you
+- [ ] Other - what other behaviors would you add?
+
+#### Follow Processes and Policy
+
+- [x] Adhere to the
+- [ ] [Your Open Source Code of Conduct]()
+- [ ] Following documented template prompts for reporting an issue or submitting
+      a PR on GitHub
+- [ ] Look for open issues of the same topic, before opening a new one.
+- [ ] Other policy
+
+## :thumbsdown: What behaviors do you want to DISCOURAGE?
+
+Remember, what's unacceptable in a business setting, is also unacceptable in
+Open Source collaborations. **This is true no matter someone's role, tenure,
+popularity or influence**. By documenting these specifically in the context of
+your project, you are setting a bar aligned with your Code of Conduct.
+
+:mega:_It's helpful to be specific, as phrases like 'Don't be a jerk' which are
+often left to interpretation of what being a jerk means._
+
+### Examples
+
+#### Disruptive Ways of Working
+
+Many Code of Conduct lists 'disruptive behavior' as a violation, which
+creates room for maintainers and community leaders alike, to identify behaviors
+that are disruptive in the context of their specific projects. There are some
+that, are easy (like no spamming), but others like the repeated re-opening of
+issues can seem to be blurry.
+
+:mega: _If behaviors contribute to continued distraction of your team, or which
+adds stress to the collaborations in your community, chances are the behavior is
+disruptive should be addressed_.
+
+- [x] Avoid use of ALL CAPS as a way of emphasis or yelling
+- [x] Don't re-open issues when sufficient attempts have been made to address
+      the concern
+- [x] No Spamming, advertising or promotion of professional services
+- [ ] What other disruptive activities would you want to include?
+
+#### Negative Impact on Community Health
+
+At the top of the many codes of conduct you'll
+see 'Our Pledge', which then lists what's often called 'protected groups', or
+'supported groups'. These groups are those who have, historically, been
+excluded, made invisible or harmed in Open Source communities. This list signals
+accountability for the experiences of the most vulnerable and marginalized
+groups in communities, and accountability to ourselves. As such, it's important
+to be on the lookout for behaviors that may target these groups.
+
+You don't need to specifically document those which seem important to highlight
```

Comment:
Not 100% certain what this line wants to get across? Maybe could be reworded to increase clarity or explicitness.